actioncable/lib/action_cable.rb

```diff
@@ -37,6 +37,10 @@ module ActionCable
     },
     default_mount_path: '/cable'.freeze
   }
+  
+  # This protocol ought to be versioned separately
+  # from the rest of the Gem
+  PROTOCOLS = ["actioncable-v1-json".freeze].freeze
```

Comment:
The protocol version is distinct from the library version and will be stable over longer periods. Using separate versions is intentional.